client-js/main/client/client.js

```diff
@@ -203,34 +216,81 @@ export class Client {
   }
 
   /**
-   * Subscribes to entity changes on the backend, providing the changes via `itemAdded`,
-   * `itemChanged`, and `itemRemoved` observers.
+   * Fetches entities of the given class type from the Spine backend. Fetches
```

Comment:
Let's shorten:
```
Fetches entities of the given class type from the Spine backend. 

Optionally accepts entity identifiers targeting the objects to fetch. If no identifiers are passed, returns all entities of the selected type.
 
The returned promise is fulfilled an array of objects, each representing an entity.

This API call is a shortcut for {@link Client#newQuery()} followed by  {@link Client#execute()}. It covers the most common use cases. If a more advanced fetch behaviour is required, the `Query` instance should be created and parameterized via {@link Client#newQuery()}.
```

Please rewrite the similar pieces in this file in the same way.